pkg/apis/config/v1alpha1/instrumentation/config.go

```diff
@@ -33,9 +35,10 @@ type Config struct {
 	// +optional
 	// +kubebuilder:example="otlp-http://localhost:4318"
 	TracingCollector string `json:"tracingCollector,omitempty"`
-	// ReportPeriod is the interval between reporting aggregated metrics.
+	// ReportPeriod is the interval between between collecting polled metrics.
```

Comment:
Final remark before merging from my behalf: would you still remove one "between" from here and regenerate?

Reply:
Should be fixed now.